### ISSUE TITLE
[BUG] Nozzle push against the nozzle wiper during cool/heating cycle of first layer inspection

### ISSUE THREAD

**Issue by thrutheframe** [bambulab/BambuStudio#231]
For some users, the nozzle pushes against the nozzle wiper during first layer inspection. This is mainly because the wiper is installed by humans on the factory grounds and there can be discrepancies. As the command for first layer inspection is hardcoded 3mf files that gets sent to the print, I would like to request that the nozzle moves a bit more to the left (1-1.5mm) to give a bit more space, just in case the installation of the nozzle wiper is not accurate enough.

thanks

**Comment by JPotter73** [comment]
I second this issue. It appears the nozzle moves into the end of the wiper, pushing the plastic support and bending the wiper assembly 
![image](https://user-images.githubusercontent.com/113537959/190935107-170683b8-8342-4fce-b63e-8a8836465f0e.png)

It happens immediately after the first layer scan while processing. The normal placement of the nozzle during purge is not being used, it seems.

My first wiper assembly came pre-bent and I replaced it, The second is installed now, but is now bent also. The second layer sometimes drags filament into the print, that I can cut off after.
Thanks!

**Comment by JPotter73** [comment]
I'd like to up vote this to a bug. My sock is being damaged by this bug. The head moves to the side of the screw on the wiper assembly at two points, I believe:

- After scanning the bed, it goes into the side of the screw to process
- After scanning the first layer, it does the same

Besides damaging the sock, this also means the wipe after completely misses the wiper and gets dragged out into the part.
![image](https://user-images.githubusercontent.com/113537959/202275542-82fb2296-21f5-4780-86d5-d214703c459d.png)

**Comment by JPotter73** [comment]
Another FB posting most likely from this bug:
https://www.facebook.com/groups/545438466970652/permalink/664698245044673/

**Comment by djngnear** [comment]
Yup I just noticed my nozzle is wiping right inbetween the PTFE and the black plastic. I circled the spot in the pic. 

![image](https://user-images.githubusercontent.com/87901234/209197358-75499320-fef7-45f1-b119-eb65afe5d3d9.jpeg)
